client/src/store/modules/quiz.js

```diff
@@ -22,6 +22,20 @@ export const quiz = {
           .catch(() => null)
         if (questions) commit('setQuizQuestions', questions)
       }
+    },
+    addQuiz ({ commit }, quiz) {
```

Comment:
if any of those does not have effect for the store it should not be in the store imo, could you move those functions to some helpers file? Actions by its origin modify the state and do not return any value

Reply:
Moved to another file in an upcoming commit